evoting/service/service.go

```diff
@@ -640,6 +672,8 @@ func new(context *onet.Context) (onet.Service, error) {
 		},
 		skipchain: context.Service(skipchain.ServiceName).(*skipchain.Service),
 	}
+	log.Lvl1("Setting skipchain propagation timeout to 5s")
+	service.skipchain.SetPropTimeout(5 * time.Second)
```

Comment:
I'm not sure if this requires a call to `skipchain.SetBFTTimeout` as well

Reply:
Let's talk about this in person and see what we can figure out.